tests/babi_test.py

```diff
@@ -1019,3 +1019,15 @@ def test_cut_uncut_multiple_file_buffers(tmpdir):
         h.await_text_missing('world')
         h.press('^U')
         h.await_text('hello\ngood\nbye\n')
+
+                         
+def test_write_and_quit(tmpdir):
+    f = tmpdir.join('f')
+
+    with run(str(f)) as h, and_exit(h):
+        h.press('hello world')
+        trigger_command_mode(h)
+        h.press_and_enter(':wq')
+        h.await_exit()
+
+    assert f.read() == 'hello world\n'
```

Comment:
can you put this next to the other command mode tests?

Reply:
Done.